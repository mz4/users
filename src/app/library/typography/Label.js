import styles from './Label.module.scss';

export const Label = ({ text }) => {
  return (
    <div className={styles.label}>
      <span>{text}</span>
    </div>
  );
};
