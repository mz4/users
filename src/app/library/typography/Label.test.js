import React from 'react';
import renderer from 'react-test-renderer';
import { Label } from './Label';

describe('Render label component', () => {
  it('Create a snapshot', () => {
    const loader = <Label />;
    const component = renderer.create(loader);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
