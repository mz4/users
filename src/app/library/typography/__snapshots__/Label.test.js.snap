// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render label component Create a snapshot 1`] = `
<div
  className="label"
>
  <span />
</div>
`;
